src/meta/Focus.ts

```diff
@@ -0,0 +1,59 @@
+import { Base } from './Base';
+import { createHandle } from '@dojo/core/lang';
+import { assign } from '@dojo/core/lang';
+import global from '@dojo/shim/global';
+
+export interface FocusResults {
+	active: boolean;
+	containsFocus: boolean;
+}
+
+const defaultResults = {
+	active: false,
+	containsFocus: false
+};
+
+export class Focus extends Base {
+	private _activeElement: Element;
```

Comment:
If another component uses `Focus` meta and sets focus, is the element set as `_activeElement` then incorrect? It looks like we set it once when it is not set, but if another meta sets focus, it doesn't get reset so the return from the `get` would be still return `true` for `node === this._activeElement`?

Reply:
Right, gotcha.